modin/backends/cudf/parser.py

```diff
@@ -20,8 +20,12 @@
 from pandas.io.common import infer_compression
 import warnings
 
+<<<<<<< HEAD
```

Comment:
These are merge conflict. Most likely you ran `git merge` and there were come conflicting errors.